virtcontainers/qemu.go

```diff
@@ -1225,12 +1226,11 @@ func genericBridges(number uint32, machineType string) []Bridge {
 	var bt bridgeType
 
 	switch machineType {
-
 	case QemuQ35:
 		// currently only pci bridges are supported
 		// qemu-2.10 will introduce pcie bridges
```

Comment:
@devimc what did you mean by 
`
		// currently only pci bridges are supported
		// qemu-2.10 will introduce pcie bridges			// qemu-2.10 will introduce pcie bridges
`

were you referring to `name "pcie-pci-bridge", bus PCI`.

If so @sboeuf as virt is based on latest qemu we should switch to pcie-pci-bridge for virt as we know it will not show up with older QEMU versions.

Reply:
@mcastelino I totally agree with you here, but again I was not trying to fix q35, but more trying to match what was already in place. IMO, we should have a follow up PR updating q35 and virt accordingly.
But your call, we can make this happen directly from this PR.